### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by MKumaran98**

### CODE REVIEW

File: src/Container/MainPage/MainPage.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import CardPage from '../CardPage/CardPage'
+import ImagePage from '../ImagePage/ImagePage'
+import InputPage from '../InputPage/InputPage'
+import ModalPage from '../ModalPage/ModalPage'; 
+import RatingPage from '../RatingPage/RatingPage';
+import LandingPage from '../LandingPage/LandingPage';
+import Hamburger from './Dropdown/Hamburger/Hamburger'
+
+const MainPage=()=>{
+    // const componentList=[
```
Comment (poojasabnani) [inline src/Container/MainPage/MainPage.js:20]: remove unnecesary code comments

File: src/Container/InputPage/TextBox/TextBox.module.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.textbox{
+    padding: 0.5rem;
+    outline: 0px;
+    border:0px;
+    letter-spacing: 1px;
+    background-color: var(--neutral-mild);
+}
```
Comment (DanishOnGit) [inline src/Container/InputPage/TextBox/TextBox.module.css]:
```suggestion
.textbox{
    padding: 0.5rem;
    outline: 0;
    border:0;
    letter-spacing: 1px;
    background-color: var(--neutral-mild);
}
```
Just refactoring

**Review by DanishOnGit — COMMENTED** [review]
Hi Muthu, 
This is very well written! Just suggested some very minor refactoring.

**Comment by ritikarp** [comment]
https://github.com/KumaranSahi/Iterative-UI/blob/69d1c46178534edbad430fe6aa3b5aac81cf9209/src/Container/LandingPage/LandingPage.js#L37

Hi Muthu, prefer adding anchor tags with icon for links than directly mentioning it.
Great work done!

**Pull request merged**